src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+spider.style.top = `${wall.clientHeight / 2 - spider.clientHeight / 2}px`;
+spider.style.left = `${wall.clientHeight / 2 - spider.clientHeight / 2}px`;
```

Comment:
Would be great to move repeatable values to variables. I mean `wall.clientHeight / 2 - spider.clientHeight / 2`. You use it twice actually